backend/tests/test_author_api.py

```diff
@@ -131,28 +135,28 @@ def test_update_githubURL(self, client, test_user, friend_user):
     def test_update_profile(self, client, test_user, friend_user):
         client.force_login(test_user)
         assert not User.objects.filter(
-            githubUrl="liluzi@gmail.com", password="somepassword").exists()
+            githubUrl="liluzi@gmail.com").exists()
         post_body_1 = json.dumps({
             "github_URL": "liluzi1@gmail.com",
             "password": "somepassword"
-
         })
 
         response = client.put("/author/update/", data=post_body_1,
                               content_type='application/json', charset='UTF-8')
         assert response.status_code == 200
-        assert not User.objects.filter(
-            githubUrl="liluzi@gmail.com", password="somepassword").exists()
+        assert not User.objects.filter(githubUrl="liluzi@gmail.com").exists()
+        author = get_object_or_404(User, username=test_user.username)
+        assert author.check_password("somepassword")
```

Comment:
no need to do `get_object`
can just do `assert test_user.check_password("somepassword")`